pkg/dashboard/apiserver/common/common.go

```diff
@@ -56,8 +56,7 @@ rules:
 - apiGroups: [""]
   resources: ["pods", "namespaces"]
   verbs: ["get", "watch", "list"]
-- apiGroups:
-  - chaos-mesh.org
+- apiGroups: ["chaos-mesh.org"]
```

Comment:
I do not get the point.

```yaml
apiGroups:
  - chaos-mesh.org
```

```yaml
apiGroups: ["chaos-mesh.org"]
```

they introduce the same meaning.

Reply:
It didn't work for me with the existing yaml until I fixed it as linked here https://kubernetes.io/docs/reference/access-authn-authz/rbac/

Reply:
A little weird but it could still work. I have no reason to block this PR.

Hi @Garima-Negi , could you append an entry into the CHANGELOG.md about this PR? ❤️

Reply:
done